FootballStats/FootballStats/Data/Source/API/APIService.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  APIService.swift
+//  FootballStats
+//
+//  Created by Hoang Linh Nguyen on 1/10/2023.
+//
+
+import UIKit
+import Alamofire
+
+final class APIService {
+    static let shared = APIService()
+
+    private init() {}
+
+    // TODO: Update [News], TeamsResponse, TopPlayersResponse, FixturesResponse
+    func request <T: Codable> (url: String, headers: HTTPHeaders, completion: @escaping (T?, String?) ->()) {
+        AF.request(url, method: .get, parameters: nil, encoding: URLEncoding.default, headers: headers)
+            .response { (response) in
+            if response.error != nil {
+                completion(nil, response.error?.localizedDescription)
+            }
+            else {
+                guard let data = response.data else {return completion(nil, response.error?.localizedDescription)}
+                let decoder = JSONDecoder()
+
+                do{
```

Comment:
sai coding convention
```suggestion
                do {
```